Gruntfile.js

```diff
@@ -292,6 +253,14 @@ var _              = require('lodash'),
                     }
                 },
 
+                jshintClient: {
+                    command: 'cd core/client && grunt jshint'
+                },
+
+                jscsClient: {
+                    command: 'cd core/client && grunt jscs'
```

Comment:
I've renamed the PR, it should be good to go now as the required client changes are in the currently referenced version.